### DISCUSSION TITLE
How to install GitLab Dracula theme on premises installation at web IDE

### DISCUSSION THREAD

**Discussion by marceloarruda01** [dracula/dracula-theme#700] [Q&A] [answered]
Hi ! I'd like to know how to install the Dracula theme on GitLab on premises. I've had add the domain to the Stylus CSS add-on configuration but besides it appears on the preferences settings, it did not work when I open de Web IDE.

If I open the file at the repository view, it works but not at Web IDE.

Thanks.

**Comment by luxonauta** [comment, marked as answer]
Hi 👋🏻, thank you for your interest in providing this feedback. ✨

But there is a better place to share the same, which is the project repository, where you can talk to the theme maintainers.

✅ [Clicking here](https://github.com/dracula/gitlab/issues/new), you can open an Issue on it.
